apps/ledger-live-mobile/src/screens/Manager/Device/DeviceName.tsx

```diff
@@ -61,6 +61,7 @@ export default function DeviceName({
         uppercase={false}
         numberOfLines={2}
         ellipsizeMode="tail"
+        flexShrink={1}
```

Comment:
"The flex-shrink CSS property sets the flex shrink factor of a flex item. If the size of all flex items is larger than the flex container, items shrink to fit according to flex-shrink" 

it's a lot of flexing and shrinking at the same time